pkgs/by-name/un/unison/package.nix

```diff
@@ -52,12 +47,12 @@ stdenv.mkDerivation (finalAttrs: {
     startupWMClass = "Unison";
   });
 
-  meta = with lib; {
+  meta = {
     homepage = "https://www.cis.upenn.edu/~bcpierce/unison/";
     description = "Bidirectional file synchronizer";
-    license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ nevivurn ];
-    platforms = platforms.unix;
+    license = lib.licenses.gpl3Plus;
+    maintainers = with lib.maintainers; [ nevivurn ];
+    platforms = lib.platforms.unix;
```

Comment:
Can you split these changes into a separate commit.

Reply:
Done, also fmt